storybook/storybook-react/src/Tabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2024 Gemeente Amsterdam
+ */
+
+import { Heading, Paragraph, Tabs } from '@amsterdam/design-system-react'
+import { Meta, StoryObj } from '@storybook/react'
+import { memo } from 'react'
+import { exampleParagraph } from '../shared/exampleContent'
+
+const SlowTab = memo(function SlowTab() {
+  // Log once. The actual slowdown is inside SlowPost.
+  console.log('[ARTIFICIALLY SLOW] Rendering 500 <SlowPost />')
+
+  let items = []
+  for (let i = 0; i < 500; i++) {
+    items.push(<SlowPost key={i} index={i} />)
+  }
+  return (
+    <ul className="items" style={{ display: 'none' }}>
+      {items}
+    </ul>
+  )
+})
+
+type SlowPostProps = {
+  index: number
+}
+
+function SlowPost({ index }: SlowPostProps) {
+  let startTime = performance.now()
+  while (performance.now() - startTime < 1) {
+    // Do nothing for 1 ms per item to emulate extremely slow code
+  }
+
+  return <li className="item">Post #{index + 1}</li>
+}
+
+const meta = {
+  title: 'Containers/Tabs',
+  component: Tabs,
+  argTypes: {
+    children: {
+      table: { disable: true },
+    },
+  },
+} satisfies Meta<typeof Tabs>
+
+export default meta
+
+const tabMeta = {
+  component: Tabs.Button,
+  args: {
+    label: 'Gegevens',
+  },
+  argTypes: {
+    label: {
+      control: { type: 'text' },
+    },
+    key: {
+      control: { type: 'number', min: 0, max: 9 },
+    },
+    isDisabled: {
+      control: { type: 'boolean' },
+    },
```

Comment:
This suggestion is a bit different from what we usually do, but: Maybe we can pass the controls for `Tabs.Button` to the first button in the Default story? That way, we would only need one story to document this component.

Reply:
This component is too complicated for our current approach of slicing stories. Let’s keep and revisit the two current examples when iterating on our documentation guidelines.